### PR TITLE
feat: allow deploy from master and main

### PR DESCRIPTION
**Pull request by cbernat**
Only two points to be modified, I will leave this pr in draft until needed. 

https://github.com/github/renaming

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -96,7 +96,7 @@ pipeline {
         }
         stage('Generate version') {
             when {
-                branch 'master'
+                branch 'main'
```
Comment (andresmoschini) [inline Jenkinsfile:99]: What do you think about keeping both temporarily? or it is completely unnecessary?

Comment (cbernat) [reply Jenkinsfile:99]: Is it possible? O_o If so I will, I was planning on leave this PR opened until needed.

Comment (andresmoschini) [reply Jenkinsfile:99]: Yep, you can generate a version when the branch is `main` or when the branch is `master`, I think that the command is _oneOf_ or something like that.

**Comment by cbernat** [comment]
@andresmoschini I´ve added optional configuration for master or main. I think I will merge this as it is, I've tried semantic release on dry-run and it seems to be working ok. What do you think?

**Pull request merged**